src/main/webapp/WEB-INF/views/users/all.jsp

```diff
@@ -0,0 +1,38 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Title</title>
+</head>
+<body>
+<h1>All users page</h1>
+
+<table border="1">
+    <tr>
+        <th>ID</th>
+        <th>Name</th>
+        <th>Login</th>
+        <th>Password</th>
+    </tr>
+    <c:forEach var="user" items="${users}">
+        <tr>
+            <td>
+                <c:out value="${user.id}"/>
+            </td>
+            <td>
+                <c:out value="${user.name}"/>
+            </td>
+            <td>
+                <c:out value="${user.login}"/>
+            </td>
+            <td>
+                <c:out value="${user.password}"/>
```

Comment:
why would you print someone's password? I don't want to buy from your shop:)

Reply:
When I was implementing it I thought that it is for admins and it is ok. But after your comment I think that you are right and admins shouldn't know my password too)